src/core/annotation.js

```diff
@@ -55,6 +55,22 @@ var ObjectLoader = coreObj.ObjectLoader;
 var FileSpec = coreObj.FileSpec;
 var OperatorList = coreEvaluator.OperatorList;
 
+function hasFlag(flags, bit) {
```

Comment:
> Can we keep these functions inside the class, but instead access them via private methods?

Of course we can, thanks for suggesting this!
I wasn't entirely happy about moving that code out of the `Annotation` class, so nice idea :-)